src/data/ua_overrides.js

```diff
@@ -1281,6 +1281,23 @@ const AVAILABLE_UA_OVERRIDES = [
       },
     },
   },
+  {
+    /*
+     * Bug 1836135 - UA override for gts-pro.sdimedia.com
+     *
+     * The site's content is not loaded without a Chrome UA spoof.
+     */
+    id: "bug1836135",
+    platform: "all",
+    domain: "www.slushy.com",
```

Comment:
Wrong domain here, this should be `gts-pro.sdimedia.com` or maybe `sdimedia.com`. Not that this has any real impact besides our listing, but.

Reply:
Good eye, will fix.